google-cloud-clients/google-cloud-bigtable-admin/src/test/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClientTest.java

```diff
@@ -303,4 +303,34 @@ public void testAwaitReplication() {
     // Verify
     assertThat(wasCalled.get()).isTrue();
   }
+
+  @Test
+  public void testExistsTrue() {
+    // Setup
+    com.google.bigtable.admin.v2.Table expectedResponse =
+                com.google.bigtable.admin.v2.Table.newBuilder()
+                    .setName(TABLE_NAME.toString())
+                    .build();
+
+    Mockito.when(mockGetTableCallable.futureCall(Matchers.any(GetTableRequest.class))).thenReturn(ApiFutures.immediateFuture(expectedResponse));
```

Comment:
Please format the code according to google java style:
http://google.github.io/styleguide/javaguide.html

You can find formatters for ides here:
https://github.com/google/styleguide